recipes/rsync/all/test_package/conanfile.py

```diff
@@ -0,0 +1,18 @@
+from conan import ConanFile
+from conan.tools.build import can_run
+import os
```

Comment:
You can remove the import for `os`.

Reply:
`os.path.join()`is used in multiple places!

Reply:
@jwillikers is right, the import `os` is not used here. The linter also notices about it. It can be removed.